Dockerfile

```diff
@@ -0,0 +1 @@
+Containerfile
```

Comment:
I am not sure what is this change.

Reply:
Hoping I am remembering correctly, when running `kubebuilder` I got some errors, because it assumes the `Dockerfile` exists and it was trying to validate it.

I solved by creating a symbolic link named `Dockerfile` pointing to the `Containerfile`. GitHub does not make it clear, but this change is a symbolic link.